test/test_in_docker.sh

```diff
@@ -20,7 +33,7 @@ function buildTestRunnerImage() {
 function deployConjur() {
   pushd ..
     git clone --single-branch \
-      --branch deploy-oss-by-default \
+      --branch deploy-oss-by-default-debug \
```

Comment:
debug. do we want it?

Reply:
k8s-conjur-deploy does not have the debug option so it makes it impossible to really located where the build fell. I think we do want this because as a 'just-in-case' measure so that if we get some weird failures from deploy (from outside our repo), we are able to see exactly where. 

I don't think this is major overhead and is convenient when a new developer starts to work on the project.